src/components/Modal/VotingPowerDelegationModal/VotingPowerDelegationCandidatesList.css

```diff
@@ -0,0 +1,82 @@
+.VotingPowerDelegationCandidatesList__Header {
+  min-height: 36px !important;
+  margin: 0 0 10px 0 !important;
+  font-size: 20px !important;
+  font-weight: 600 !important;
+  line-height: 36px !important;
+}
+
+.VotingPowerDelegationCandidatesList__Description {
+  margin: 0 0 32px 0 !important;
+}
+
+.VotingPowerDelegationCandidatesList > .dcl.close,
+.ui.modal > .dcl.close {
+  top: 22px; /* TODO: Can this be a property to override in all modals? */
```

Comment:
if you want to add this to all modals, you can add it in theme.css I think

Reply:
I'm not sure about this, I just moved the CSS with the comment, so I'll leave it there for now